src/index.ts

```diff
@@ -110,6 +112,11 @@ export function options(config: CodegenConfig, context: SwiftGeneratorContext):
 		package: {
 			name: packageName,
 		},
+		logging: {
+			enabled: configBoolean(logging, 'enabled', false, 'logging.'),
+			subsystem: configString(logging, 'subsystem', "openapi.generator.plus", 'logging.'),
+			category: configString(logging, 'category', "openapi", 'logging.'),
```

Comment:
I'm not sure what to make the category then...

Maybe category should default to the `package`, then the subsystem is "api" and the category is "com.letterboxd.api" or something like that. By default.

I've revised this below to say let's get rid of this.